custom_components/ha_creality_ws/utils.py

```
@@ -330,4 +337,270 @@ def resolved_model(self) -> str:
+        if value is not None and str(value).strip():
+            payload[key] = str(value).strip()
+
+    if color is not None:
+        payload["color"] = normalize_material_color(color)
+
+    low = safe_float(min_temp)
+    high = safe_float(max_temp)
+    if low is not None and high is not None and high < low:
+        raise ValueError(
+            f"max_temp ({high}) must not be below min_temp ({low})"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)